### PR TITLE
Eloquent meshes

### PR DESCRIPTION
**Pull request by nbbrooks**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,198 +1,255 @@
...
-else()
-  find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
-  set(QT_LIBRARIES Qt5::Widgets)
-  macro(qt_wrap_ui)
-    qt5_wrap_ui(${ARGN})
-  endmacro()
-endif()
-## Prefer the Qt signals and slots to avoid defining "emit", "slots",
-## etc because they can conflict with boost signals, so define QT_NO_KEYWORDS here.
+# if("${rviz_QT_VERSION}" VERSION_LESS "5")
```
Comment (davetcoleman) [inline CMakeLists.txt]: For ROS 2 we don't need support of pre-QT5, just remove this commented section

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,198 +1,255 @@
...
-  src/key_tool.cpp
-)
+include_directories(
+  include
+  ${Eigen3_INCLUDE_DIRS})
 
 # Rviz GUI library
-add_library(${PROJECT_NAME}_gui ${SOURCE_FILES} src/remote_control.cpp)
-target_link_libraries(${PROJECT_NAME}_gui ${rviz_DEFAULT_PLUGIN_LIBRARIES} ${QT_LIBRARIES} ${catkin_LIBRARIES})
+# set(SOURCE_FILES
```
Comment (davetcoleman) [inline CMakeLists.txt]: Add TODO to convert to ROS 2

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,198 +1,255 @@
...
+##########
+
+if (BUILD_TESTING)
+  # # Lint
+  # find_package(ament_lint_auto REQUIRED)
+  # ament_lint_auto_find_test_dependencies()
+
+  # # Run tests
+  # find_package(ament_cmake_gtest REQUIRED)
+  # ament_add_gtest(${PROJECT_NAME}_rvt_test tests/rvt_test.cpp)
```
Comment (davetcoleman) [inline CMakeLists.txt]: Do not disable tests! I spent a lot of time on this and its how you verify your port works...

**Comment by mlautman** [comment]
@nbbrooks Clang tidy is failing because of the STL's Can you look into why and fix?

**Comment by nbbrooks** [comment]
Going to make a fresh PR to try and fix LFS issues

**Pull request closed without merging**